lib/iotagent-json.js

```diff
@@ -185,5 +185,24 @@ function stop(callback) {
     );
 }
 
+/**
+ * Shuts down the IoT Agent in a graceful manner
+ *
+ */
+function handleShutdown(signal) {
+    config.getLogger().fatal(context, 'Received %s, starting shutdown processs', signal);
```

Comment:
Not sure 'fatal' would be the right one in this case... The 'fatal' loglevel is though for very critical fails and a ordered shutdown is not of that kind. Maybe 'info' is better.

@AlvaroVega @mapedraza what do you think?

Reply:
You could get this SIGINT when running on Kubernetes. So it could be an outside process causing the IoT Agent to fail

Reply:
Note Orion is using INFO level for the same purpose:

https://github.com/telefonicaid/fiware-orion/blob/78db2d16ff7052cf8e46c0db4c69dc106172fc3a/src/app/contextBroker/contextBroker.cpp#L530

Reply:
Info is enough from my point of view

Reply:
Fixed b86a366